### PR TITLE
Update for changes with CollisionMarginData

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

File: trajopt_utils/package.xml
Comment (mpowelson) [inline trajopt_utils/package.xml:16]: Does this need the -dev version?

**Review by mpowelson — APPROVED** [review]

File: trajopt_utils/package.xml
Comment (Levi-Armstrong) [reply trajopt_utils/package.xml:16]: Good catch. I updated it to the following but going to wait for windows CI to finish because it is failing for some reason related to osqp with an odd error.

``` xml
  <build_depend>libboost-program-options-dev</build_depend>
  <exec_depend>libboost-program-options</exec_depend>
```

**Comment by Levi-Armstrong** [comment]
I will merge if the windows build fails again. It look like something must have change upstream in ros-win and will need more time to look into what changed.

**Pull request merged**
